src/components/layout/sidenav/index.tsx

```diff
@@ -21,7 +21,7 @@ const SideNav = () => {
       >
         <OmnexIcon />
       </div>
-      <div className="flex flex-col justify-center items-center gap-[10px] overflow-y-auto">
```

Comment:
why overflow-y-auto removed?

Reply:
As a result of utilizing 'overflow-y-auto', an unintended vertical scrolling issue is occurring specifically on Mac screens.